components/encumbered/lms/Makefile

```diff
@@ -0,0 +1,119 @@
+#
+# CDDL HEADER START
+#
```

Comment:
Please, use new copyright header (look for example at templates/justmake.mk)

Reply:
I did not know there is a new one. Sorry.

Reply:
Took the one from justmake.mk.